packages/comby/comby.1.7.0/opam

```diff
@@ -0,0 +1,62 @@
+opam-version: "2.0"
+maintainer: "rvantonder@gmail.com"
+authors: "Rijnard van Tonder"
+homepage: "https://github.com/comby-tools/comby"
+bug-reports: "https://github.com/comby-tools/comby/issues"
+dev-repo: "git+https://github.com/comby-tools/comby.git"
+license: "Apache-2.0"
+build: [
+    [
+      "dune"
+      "build"
+      "-p"
+      name
+      "-j"
+      jobs
+      "@install"
+      "@runtest" {with-test}
+    ]
+]
+depends: [
+  "dune" {>= "2.8.0"}
+  "ocaml" {>= "4.08.1"}
+  "conf-autoconf"
+  "conf-gmp"
+  "conf-libev" {os-distribution != "ol"}
+  "conf-libpcre"
+  "conf-m4" {os = "linux"}
+  "conf-pkg-config"
+  "conf-sqlite3"
+  "conf-zlib" {os = "linux"}
+  "cohttp-lwt-unix"
+  "comby-kernel" {= "1.7.0"}
```

Comment:
For next time know that you can replace `{= "1.7.0"}` by `{= version}` for simplicity.

Reply:
Niiice, will do!